src/espresso.js

```diff
@@ -26,6 +26,7 @@ export default class Espresso {
     this.reporter = options.reporter;
     this.server = new Server();
     this.mocha = new MochaParallel();
+    this.port = options.port || 8545;
```

Comment:
Instead of storing this on the `Espresso` instance, can you store it on the `Server` instance? So pass it into the `Server` constructor.